components/viewProduct/ToyPreview.tsx

```diff
@@ -90,40 +68,40 @@ const StFillToyMark = styled(IcFillToyMark)`
   left: 0.2rem;
 `;
 const StStore = styled.div`
-  width: 22.4rem;
-  margin-top: 1.6rem;
-  font-weight: 500;
-  font-size: 1.4rem;
-  color: #a9a9a9;
+  width: 22rem;
+  height: 2rem;
+  margin-top: 1.2rem;
+
+  font: ${({ theme }) => theme.fonts.b5_14_medium_140};
+  color: ${({ theme }) => theme.colors.gray006};
 `;
 
 const StTitle = styled.div`
-  width: 22.4rem;
-  margin-top: 0.8rem;
-  font-weight: 400;
-  font-size: 1.6rem;
-  display: flex;
-  align-items: center;
-  color: #000000;
+  width: 22rem;
+  height: 4.4rem;
+  margin-top: 0.4rem;
+
+  font: ${({ theme }) => theme.fonts.b3_16_medium_140};
 `;
 
 const StPrice = styled.div`
-  width: 22.4rem;
-  margin-top: 1.3rem;
-  font-weight: 700;
-  font-size: 1.9rem;
+  width: 22rem;
+  height: 2.5rem;
+  margin-top: 0.4rem;
+
+  font: ${({ theme }) => theme.fonts.b2_18_bold_140};
+
```

Comment:
오 font 이렇게 사용안하고 밑에처럼 작성해도 먹혀! 내가 그러케 만들어나써! color랑달라영!

```suggestion
${({ theme }) => theme.fonts.b2_18_bold_140};
```